sdk/communication/AzureCommunicationCalling/AzureCommunicationCalling-Swift.h

```diff
@@ -0,0 +1,1027 @@
+    case captureAdapter = 2
+
+    /// Virtual Video Device
+    case virtual = 3
+}
+
+/// Local and Remote Video Stream types
+public enum MediaStreamType : Int {
+
+    
+    /// Video
+    case video = 0
+
+    /// Screen share
+    case screenSharing = 1
+}
+
+/// State of a participant in the call
+public enum ParticipantState : Int {
+
+    
+    /// Idle
+    case idle = 0
+
+    /// Early Media
+    case earlyMedia = 1
+
+    /// Connecting
+    case connecting = 2
+
+    /// Connected
+    case connected = 3
+
+    /// On Hold
+    case hold = 4
+
+    /// In Lobby
+    case inLobby = 5
+
+    /// Disconnected
+    case disconnected = 6
+
+    /// Ringing
+    case ringing = 7
+}
+
+/// State of a call
+public enum CallState : Int {
+
+    
+    /// None - disposed or applicable very early in lifetime of a call
+    case none = 0
+
+    /// Early Media
+    case earlyMedia = 1
+
+    /// Call is being connected
+    case connecting = 3
+
+    /// Call is ringing
+    case ringing = 4
+
+    /// Call is connected
+    case connected = 5
+
+    /// Call held by local participant
+    case localHold = 6
+
+    /// Call is being disconnected
+    case disconnecting = 7
+
+    /// Call is disconnected
+    case disconnected = 8
+
+    /// In Lobby
+    case inLobby = 9
+
+    /// Call held by a remote participant
+    case remoteHold = 10
+}
+
+/// Direction of a Call
+public enum CallDirection : Int {
+
+    
+    /// Outgoing call
+    case outgoing = 1
+
+    /// Incoming call
+    case incoming = 2
+}
+
+/// DTMF (Dual-Tone Multi-Frequency) tone for PSTN calls
+public enum DtmfTone : Int {
+
+    
+    /// Zero
+    case zero = 0
+
+    /// One
+    case one = 1
+
+    /// Two
+    case two = 2
+
+    /// Three
+    case three = 3
+
+    /// Four
+    case four = 4
+
+    /// Five
+    case five = 5
+
+    /// Six
+    case six = 6
+
+    /// Seven
+    case seven = 7
+
+    /// Eight
+    case eight = 8
+
+    /// Nine
+    case nine = 9
+
+    /// Star
+    case star = 10
+
+    /// Pound
+    case pound = 11
+
+    /// A
+    case A = 12
+
+    /// B
+    case B = 13
+
+    /// C
+    case C = 14
+
+    /// D
+    case D = 15
+
+    /// Flash
+    case flash = 16
+}
+
+/// Local and Remote Video scaling mode
+public enum ScalingMode : Int {
+
+    
+    /// Cropped
+    case crop = 1
+
+    /// Fitted
+    case fit = 2
+}
+
+public enum HandleType : Int {
+
+    
+    case unknown = 0
+
+    case groupCallLocator = 1
+
+    case teamsMeetingCoordinatesLocator = 2
+
+    case teamsMeetingLinkLocator = 3
+}
+
+// MARK: Forward declarations.
+
+
+/**
+ * A set of methods that are called by ACSCallAgent in response to important events.
+ */
+public protocol CallAgentDelegate : NSObjectProtocol {
+
+    
+    optional func callAgent(_ callAgent: CallAgent!, didUpdateCalls args: CallsUpdatedEventArgs!)
+
+    optional func callAgent(_ callAgent: CallAgent!, didRecieveIncomingCall incomingcall: IncomingCall!)
```

Comment:
Looks good though `incomingcall` should probably be `incomingCall`.

Reply:
Done will change it

Reply:
Done changed